engine/apps/mobile_app/tasks.py

```diff
@@ -47,38 +58,76 @@ def notify_user_async(user_pk, alert_group_pk, notification_policy_pk, critical)
             notification_step=notification_policy.step,
             notification_channel=notification_policy.notify_by,
         )
-        logger.info(f"Error while sending a mobile push notification: user {user_pk} has no devices set up")
+        logger.info(f"Error while sending a mobile push notification: user {user_pk} has no device set up")
         return
 
     message = get_push_notification_message(alert_group)
     thread_id = f"{alert_group.channel.organization.public_primary_key}:{alert_group.public_primary_key}"
+    alert_title = f"Critical page: {message}" if critical else message
+    number_of_alerts = alert_group.alerts.count()
 
-    if critical:
-        aps = {
-            "alert": f"Critical page: {message}",
-            "interruption-level": "critical",
-            "sound": "ambulance.aiff",
-        }
-    else:
-        aps = {
-            "alert": message,
-            "sound": "bingbong.aiff",
-        }
+    # TODO: we should update this to check if FCM_RELAY is set and conditionally make a call here..
 
-    extra = {
-        "orgId": alert_group.channel.organization.public_primary_key,
-        "orgName": alert_group.channel.organization.stack_slug,
-        "alertGroupId": alert_group.public_primary_key,
-        "status": alert_group.status,
-        "aps": aps,
-    }
+    message = Message(
```

Comment:
this was the main focus of this PR, adjusting the push notification settings to include some iOS and Android specific things. `django-push-notifications` didn't allow this because it was using the "legacy" FCM API whereas `fcm-django` uses the latest API, which supports these features.